src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const shortTerm = 3;
+  const longTerm = 7;
+  const price = 40;
```

Comment:
do it right away to avoid repeating in the code bellow

```suggestion
  const price = 40 * days;
```